household-esignet-integration-impl/README.md

```diff
@@ -1,5 +1,54 @@
-# mock-esignet-integration-impl
+# HouseHold ID e-Signet plugin implementation
 
 ## About
+e-Signet integration with householdID view. Here the householdID DB view plays the role of ID repository. IDA implementation to identify users based on the provided ID number is part of the authenticator plugin.
 
-Implementation for all the interfaces defined in esignet-integration-api.
\ No newline at end of file
+KYC returned as part of userinfo response will only contain the subject claim. Subject claim will hold the household ID.
+
+Prerequisite:
+1. Insert new key alias, in key_alias table, this key is used to sign the KYC
+> INSERT INTO KEY_POLICY_DEF(APP_ID,KEY_VALIDITY_DURATION,PRE_EXPIRE_DAYS,ACCESS_ALLOWED,IS_ACTIVE,CR_BY,CR_DTIMES) VALUES('HH_AUTHENTICATION_SERVICE', 1095, 50, 'NA', true, 'mosipadmin', now());
+
+2. Update below keys in the esignet properties file
+> mosip.esignet.integration.scan-base-package=io.mosip.householdid.esignet.integration
+>
+> mosip.esignet.integration.authenticator=HouseHoldAuthenticationService
+>
+> mosip.esignet.integration.key-binder=HouseHoldKeyBinder
```

Comment:
Keybinder is supported for HHID ?

Reply:
Nope it is not, we throw notimplemented exception